temba/flows/views.py

```diff
@@ -578,6 +620,15 @@ def __init__(self, user, *args, **kwargs):
                     )
 
                     self.fields[Flow.CONTACT_CREATION] = contact_creation
+                elif self.instance.flow_type == Flow.USSD:
+                    ussd_trigger = Trigger.objects.filter(
+                        org=self.instance.org, flow=self.instance, is_archived=False, groups=None,
+                        trigger_type=Trigger.TYPE_USSD_PULL
+                    ).first()
+                    if ussd_trigger:
+                        self.fields['ussd_trigger'].initial = ussd_trigger.keyword
+                        self.fields['ussd_trigger'].widget.attrs['readonly'] = True
+                        self.fields['ussd_trigger'].help_text = 'To change the USSD trigger code, go to Triggers'
```

Comment:
@ericnewcomer okay, should I remove it from create & update too or only from update ?